framework/src/executioners/Steady.C

```diff
@@ -36,6 +36,8 @@ Steady::init()
   checkIntegrity();
   _problem.execute(EXEC_PRE_MULTIAPP_SETUP);
   _problem.initialSetup();
+  _fixed_point_solve->initialSetup();
+  _feproblem_solve.initialSetup();
```

Comment:
so the executioner is in charge of calling initial setup on the problem and the solve objects.

are we doing the same for solve() / postSolve() / ?

Reply:
The only other option I see is making `Executioner::initialSetup()` and then calling that in `FEProblemBase::initialSetup()`, since the executioner owns the solves.

Reply:
Appears consistent to me.